fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -192,7 +193,7 @@ public Triple apply(final Statement input) {
         }
         servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
-        return Response.ok(rdfStream).build();
+        return ok(rdfStream.withThisContext(rdfStream.transform(new Deskolemizer(idTranslator, null)))).build();
```

Comment:
Hm. Very good point. Let me see if I can move this deeper, out of HTTP. I think it very likely that I can.